perun-openapi/openapi.yml

```diff
@@ -10658,3 +10676,25 @@ paths:
           $ref: '#/components/responses/VoidResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
+
+  #################################################
+  #                                               #
+  # RTMessagesManager                               #
+  #                                               #
+  #################################################
+
+  /urlinjsonout/rtMessagesManager/sentMessageToRT:
```

Comment:
This method has more possible parameter combinations. Please, suffix it and add a TODO stating that there has to be implemented other version of the method.